pyfar/dsp/dsp.py

```diff
@@ -187,3 +187,119 @@ def spectrogram(signal, dB=True, log_prefix=20, log_reference=1,
     times -= times[0]
 
     return frequencies, times, spectrogram
+
+
+def normalize(signal, normalize='time', normalize_to='max',
+              channel_handling='max', value=None, freq_range=None,
+              return_values=False):
+    """
+    Normalize signal in the time or frequency domain.
+
+    Parameters
+    ----------
+    signal: Signal
+        Input signal of the signal class
+    normalize: string
+        'time' - Normalize the time signal to 'value'
+        'magnitude' - Normalize the magnitude spectrum to 'value'
```

Comment:
- you could add a `power` option to be consistent with `average`
- to avoid duplicate code, you can have a private function that gives you the desired signal representation (time, magnitude, ...). This function can be used in average and normalize